### PR TITLE
MAINT: Create FUNDING.yml

### PR DESCRIPTION
**Pull request by Gui-FernandesBR**
## Pull request type

- [X] ReadMe, Docs and GitHub maintenance

## Pull request checklist

Please check if your PR fulfills the following requirements, depending on the type of PR:

- ReadMe, Docs and GitHub maintenance:

  - [X] Spelling has been verified
  - [X] Code docs are working correctly 

- Code base maintenance (refactoring, formatting, renaming):

## What is the new behavior?

Now adding a new Sponsors Button on our main page

## Does this introduce a breaking change?

- [X] No

## Other information

In the future we probably may publish sponsors logos here

### CODE REVIEW

**Comment by giovaniceotto** [comment]
Should we create other sponsor accounts, such as Patreon?

File: .github/FUNDING.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+patreon: # Replace with a single Patreon username
+open_collective: # Replace with a single Open Collective username
+ko_fi: # Replace with a single Ko-fi username
+tidelift: # Replace with a single Tidelift platform-name/package-name e.g., npm/babel
+community_bridge: # Replace with a single Community Bridge project-name e.g., cloud-foundry
+liberapay: # Replace with a single Liberapay username
+issuehunt: # Replace with a single IssueHunt username
+otechie: # Replace with a single Otechie username
+lfx_crowdfunding: # Replace with a single LFX Crowdfunding project-name e.g., cloud-foundry
+custom: # Replace with up to 4 custom sponsorship URLs e.g., ['link1', 'link2']
```
Comment (giovaniceotto) [inline .github/FUNDING.yml:13]:
```suggestion
```

**Review by giovaniceotto — CHANGES_REQUESTED** [review]
Please remove the unused model platforms to prevent syntax erros in the yaml file.

**Review by giovaniceotto — APPROVED** [review]

**Comment by Gui-FernandesBR** [comment]
> Should we create other sponsor accounts, such as Patreon?

Sure thing! Let's note this for later

**Pull request merged**